components/src/core/components/Input/Select/select-input.scss

```diff
@@ -220,23 +219,51 @@
   border-radius: 0.85rem;
   &.button-double-line {
     padding: 0.5rem 0.75rem 0.5rem 1rem;
+    :deep(.oxd-button-label-wrapper) {
```

Comment:
Why do we still need this here? Can't we remove it?

Reply:
Removed all the styles related to select input button component